_docs/ops/contingency-plan.md

```diff
@@ -130,6 +130,17 @@ If all AWS regions are disrupted, Cloud Operations will deploy the system to ano
 #### AWS RDS
 Major services (Cloud Foundry, BOSH, Concourse, and UAA) rely on databases provided by RDS. To restore a database from backup, see [restoring RDS]({{ site.baseurl }}{% link _docs/ops/runbook/restoring-rds.md %}).
 
+#### Brokered AWS services
+
+cloud.gov is not dependent on the following services, but we do broker customer provisioning and access to these services, so our customers could be impacted if they were unavailable.  
+
+If a GovCloud service were wholly unavailable, we would broker access to the AWS commercial cloud equivalent, as shown below:
```

Comment:
Are we truly able to do this?

Reply:
There would be engineering work regarding the authorization of the broker (across account boundaries) and the services would no longer be in the same logical VPC, so we'd have to make the call whether our customers can accept those risks, and communicate those clearly.

Reply:
Okay, that sounds reasonable; do we also need to make a note about we'd have to receive approval from our AO for this change to occur such as the point above about redeploying in another IaaS environment, or is that out of scope for this?